packages/bits/src/lib/freetype-query-builder/freetype-query-builder.component.less

```diff
@@ -0,0 +1,81 @@
+@import (reference) "../../styles/nui-framework-variables.less";
+@import (reference) "../../styles/mixins.less";
```

Comment:
why we still use relative path for style mixins?

Reply:
I just wanted to keep import styles based on existing components. Fixed.